Gopkg.toml

```diff
@@ -33,7 +33,7 @@ required = [
 
 [[constraint]]
   name = "k8s.io/kops"
-  version = "1.9.1"
+  branch = "master"
```

Comment:
Would the plan then be to change this to a specific version again once there is a tag that includes the change?

Reply:
Yes. There are multiple other packages in very much the same situation, but I wouldn't make it a high priority just yet...